committeeoversightapp/forms.py

```diff
@@ -1,45 +1,36 @@
 from django.forms import ModelForm, Form, TextInput, HiddenInput, DateField, DateInput, \
-                         ModelMultipleChoiceField, ChoiceField, CharField, Textarea
+                         ModelChoiceField, ModelMultipleChoiceField, ChoiceField, \
+                         CharField, formset_factory
 from opencivicdata.legislative.models import Event, EventParticipant
 from opencivicdata.core.models import Jurisdiction, Organization
+
+from .models import HearingCategoryType
 from .customfields import GroupedModelMultiChoiceField
 
+
 # add hearing events
 class EventForm(ModelForm):
     class Meta:
         model = Event
-        fields = ['jurisdiction', 'name', 'start_date', 'classification', 'status']
+        fields = ['jurisdiction', 'name', 'start_date', 'classification']
         labels = {
             'name': ('Hearing Title'),
-            'start_date': ('Date')
         }
         widgets = {
-            'jurisdiction': HiddenInput()
+            'jurisdiction': HiddenInput(),
+            'classification': HiddenInput()
             }
 
     start_date = DateField(
-        widget=DateInput(attrs={'placeholder': 'yyyy-mm-dd'})
+        widget=DateInput(attrs={'placeholder': 'yyyy-mm-dd'}),
```

Comment:
Not terribly important at the moment, but you might look into using [jquery's datepicker](https://jqueryui.com/datepicker/) for this field.

Reply:
On my list of questions for Lugar is whether or not we can depend on this always being an exact day!

Reply:
Opening an issue + closing